pkg/orchestrator/redact/pod.go

```diff
@@ -53,3 +55,11 @@ func ScrubContainer(c *v1.Container, scrubber *DataScrubber) {
 		c.Args = scrubbedMergedCommand[words:]
 	}
 }
+
+// RemoveLastAppliedConfigurationAnnotation redacts the whole "kubectl.kubernetes.io/last-applied-configuration" annotation. As it may contain duplicate information and secrets.
+func RemoveLastAppliedConfigurationAnnotation(annotations map[string]string) {
+	a := annotations["kubectl.kubernetes.io/last-applied-configuration"]
```

Comment:
are we sure this key `kubectl.kubernetes.io/last-applied-configuration` always exist?

Reply:
If the value doesn't exist the map access will return "" and if it exist and is set to "" we don't have to change it as well. I will add a unit test for this 👍 . If you think it is better I can make it more explicit:
```
	if val, ok := annotations["kubectl.kubernetes.io/last-applied-configuration"]; ok{
		if val != "" {
			annotations["kubectl.kubernetes.io/last-applied-configuration"] = replacedValue
		}
	}
}
```

I have no big opinion here. I do like the current implementation a tad more.

Reply:
my bad, I think I did too much Python to be traumatized with unsafe dict access but in Go case we're good 👍

Reply:
Thinking about this again. I agree, let me go the idiomatic way and make sure we always redact if it exists, even if empty.